<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smoothingClass</class>
 <widget class="QDialog" name="smoothingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>460</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>381</width>
     <height>421</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>419</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>381</width>
     <height>421</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>419</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>GaussianBlur( )</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Compute</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ksize</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>sigmaX</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>sigmaY</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>borderType</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider">
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>medianBlur( )</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>ksize</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Compute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>AverageBlur( )</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ksize</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>anchorX</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>anchorY</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>borderType</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_5">
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Compute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_gaussianEnter_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_gaussian_ksize_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_ksize_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_restore_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>smoothingClass</receiver>
   <slot>test2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_sigmaX_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_sigmaY_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_borderType_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_sigmaY_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_sigmaX_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_gaussianSend_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_medianEnter_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_medianSend_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_averageSend_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>smoothingClass</receiver>
   <slot>on_averageEnter_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_gaussian_ksize_clicked()</slot>
  <slot>on_sigmaX_clicked()</slot>
  <slot>on_sigmaY_clicked()</slot>
  <slot>on_borderType_clicked()</slot>
  <slot>on_gaussianEnter_clicked()</slot>
  <slot>on_restore_clicked()</slot>
  <slot>slot1()</slot>
  <slot>on_gaussianSend_clicked()</slot>
  <slot>on_medianEnter_clicked()</slot>
  <slot>on_medianSend_clicked()</slot>
  <slot>on_averageSend_clicked()</slot>
  <slot>on_averageEnter_clicked()</slot>
 </slots>
</ui>
